.vortex/tooling/src/helpers.php

```
@@ -549,6 +644,116 @@ function lagoon_exec(string $bin, string $subcommand, array $ctx, ?int &$exit_co
+
+  task(sprintf('Downloading the Acquia CLI "%s" to "%s".', $version, $bin));
+  $response = request($url, ['method' => 'GET', 'save_to' => $bin, 'timeout' => 120]);
+  if (!$response['ok']) {
+    @unlink($bin);
+    fail('Failed to download the Acquia CLI from "%s": %s', $url, $response['error'] ?? 'Unknown error');
+  }
+
+  chmod($bin, 0755);
+
+  return $bin;
+}
```

PHPMD (2.15.0)

[error] Remove error control operator '@' on line 683. (undefined)

(ErrorControlOperator)